src/main/features/response/routes/receipt.ts

```diff
@@ -1,39 +1,43 @@
 import * as express from 'express'
```

Comment:
Shouldn't we rename the file to be `response-receipt.ts`? What would match change you did for claim where you renamed route to be `claim-receipt.ts`.

Reply:
Changing now

Reply:
I would prefer it to be `receipt` or `pdf` it is in the response journey no need to prefix it

Reply:
Its receipt.ts now (after being changed to response-receipt)